apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.tsx

```diff
@@ -252,14 +271,38 @@ const CaseFile: React.FC<CaseFileProps> = (props) => {
                   key={`${caseFile.id}-edit`}
                 >
                   <Box display="flex">
-                    <Box flexGrow={1} marginRight={2}>
-                      <Input
-                        name="fileName"
-                        size="xs"
-                        placeholder={formatMessage(m.simpleInputPlaceholder)}
-                        defaultValue={displayName}
-                        onChange={(evt) => setEditedFilename(evt.target.value)}
-                      />
+                    <Box className={styles.editCaseFileInputContainer}>
+                      <Box className={styles.editCaseFileName}>
+                        <Input
+                          name="fileName"
+                          size="xs"
+                          placeholder={formatMessage(m.simpleInputPlaceholder)}
+                          defaultValue={displayName}
+                          onChange={(evt) =>
+                            setEditedFilename(evt.target.value)
+                          }
+                        />
+                      </Box>
+                      <Box className={styles.editCaseFileDisplayDate}>
+                        <InputMask
+                          mask={'99.99.9999'}
+                          maskPlaceholder={null}
+                          value={editedDisplayDate || ''}
+                          onChange={(evt) => {
+                            setEditedDisplayDate(evt.target.value)
+                          }}
+                          onBlur={(evt) => {
```

Comment:
Is this needed since we have onChange that does the same thing?

Reply:
I guess it is not needed. I just copy-pasted from other use-cases of `InputMask`